src/python/pants/backend/typescript/dependency_inference/rules.py

```diff
@@ -10,8 +10,11 @@
 from pants.backend.javascript import package_json
 from pants.backend.javascript.dependency_inference.rules import (
     InferNodePackageDependenciesRequest,
-    NodePackageCandidateMap,
     RequestNodePackagesCandidateMap,
+    _determine_import_from_candidates,
+    _handle_unowned_imports,
+    _is_node_builtin_module,
```

Comment:
Maybe these functions should become public, not `_`?

Reply:
Hm, potentially, since I am importing them in a different backend (albeit the JS and TS backends are so tightly coupled). I think, however, since we don't have a clear public API, at least marking those functions as internal should give a hint that we don't want them to be usable in plugins? So perhaps leaving as private would still make sense? :/

Reply:
Sounds good to me, thanks for explaining!